components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -59,21 +61,10 @@ - (void)commonMDCInputTextFieldInit {
 #pragma mark View Setup
 
 - (void)initializeProperties {
-  [self setUpLayoutDirection];
-  [self setUpLabelBehavior];
-  [self setUpLabelState];
-}
-
-- (void)setUpLayoutDirection {
   self.layoutDirection = self.mdf_effectiveUserInterfaceLayoutDirection;
-}
-
-- (void)setUpLabelBehavior {
   self.labelBehavior = MDCTextControlLabelBehaviorFloats;
-}
-
-- (void)setUpLabelState {
   self.labelState = [self determineCurrentLabelState];
```

Comment:
Solid cleanup, but please send a separate PR for it.

Reply:
Meh I'll just put it back the way it was

Reply:
If the overhead of making a PR is feeling too high please let me know, it should feel super easy to knock out a solid small cleanup PR like this.